packages/manager/apps/web/client/app/hosting/cron/hosting-cron.service.js

```diff
@@ -89,9 +100,11 @@ export default class HostingCron {
           serviceName,
         },
       },
-    ).then((data) => {
-      this.Hosting.resetCrons();
-      return data;
+    ).then((response) => {
+      if (response.state !== 'ERROR') {
+        this.pollingActions(serviceName, 'cron/delete');
```

Comment:
Nitpick: can you include service name in namespace (like 'web/hosting/cron/delete') of polling to avoid coalition with other tasks from other services?